docs/README.md

```diff
@@ -0,0 +1,214 @@
+# Integrating Kobiton into CircleCI mobile application development pipeline
+## Table of contents
+- [A. Integrating Kobiton with CircleCI](#a-integrating-kobiton-with-circleci)
+    - [1. Preparation](#1-preparation)
+    - [2. Setup](#2-setup)
+        - [2.1. Setting Kobiton Username and API Key](#21-setting-kobiton-username-and-api-key)
+        - [2.2. Setting Kobiton device desired capabilities](#22-setting-kobiton-device-desired-capabilities)
+        - [2.3. CircleCI configuration file](#23-circleci-configuration-file)
+    - [3. Automation Test Execution](#3-automation-test-execution)
+- [B. Test session details](#b-test-session-details)
+    - [1. Viewing test session details on Kobiton website](#1-viewing-test-session-details-on-kobiton-website)
+    - [2. Fetching test session details using Kobiton REST API](#2-fetching-test-session-details-using-kobiton-rest-api)
+- [C. Feedback](#c-feedback)
+
+## A. Integrating Kobiton with CircleCI
+### 1. Preparation
+#### 1.1. Getting Kobiton Username and API key
+Kobiton Username and API key are required for authenticating with Kobiton.
```

Comment:
just my suggestion, maybe shorten to "Username and API key... with Kobiton" or "Kobiton Username... authenticating".
@NguyenHoanTien hoan
@blackheat